lang/cs/Org.Apache.REEF.IO.TestClient/Org.Apache.REEF.IO.TestClient.csproj

```diff
@@ -0,0 +1,114 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think we should create this new project. As-is we already have too many projects. This should go in Org.Apache.REEF.IO. Not sure why we are creating a new one.